src/H5VL.c

```diff
@@ -654,6 +654,37 @@ H5VLobject(hid_t id)
     FUNC_LEAVE_API(ret_value)
 } /* H5VLobject() */
 
+/*---------------------------------------------------------------------------
+ * Function:    H5VLobject_is_native
+ *
+ * Purpose:     Determines whether an object ID represents a native VOL
+ *              connector object.
+ *
+ * Return:      Success:    TRUE/FALSE
+ *              Failure:    FAIL
```

Comment:
We need to fix this! This API call needs to be reworked so it returns herr_t and hbool_t is_native is an out parameter (htri_t is trash and should not be used in new code).

Reply:
It was changed 5 days later.  I've pulled that change in, please review again.